openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -108,13 +108,3 @@ def get_tabs(self, course_overview):
                 'url': tab.link_func(course_overview, reverse),
             })
         return tabs
-
-    def get_enrollment(self, course_overview):
```

Comment:
Because the enrollment field above changed to a `DictField`, does that mean this method is no longer necessary?  Is the change backwards compatible for whatever was using this method?

Reply:
Yes. `SerializerMethodField` means "get the result by calling the method `get_enrollment`". Since I refactored where the enrollment is coming from, it's not used anymore. The response is unchanged, though.